models/article-models.js

```diff
@@ -13,20 +13,28 @@ function fetchArticlesByID(article_id){
     })
 }
 
-function fetchAllArticles(order = "DESC", sort_by = "created_at"){
-    
+function fetchAllArticles(order = "DESC", sort_by = "created_at", topic){    
     const allowedOrderQueries = ["DESC","ASC"]
 
     const allowedSortByQueries = ["title", "created_at", "topic", "author", "votes"]
 
+    const allowedTopicQueries = ["mitch", "cats", "paper", undefined]
```

Comment:
I would use format for this because eventually you are going to have so many topics and it would be too much work to come back to greenlist all of them